server/views/pages/profile.html

```diff
@@ -172,9 +172,11 @@ <h2 class="govuk-heading-m">Visits</h2>
                   {%- endcall %}
                 </li>
               {%- endif %}
-              {# <li class="govuk-grid-column-one-quarter card-group__item">
-                {{imageCard({ id: 'approvedVisitors', src: "/public/images/image_visitors.jpg", alt: "visitor picture", link: knownPages.profile.visitors, description: "View your full list of approved visitors" }) }}
-              </li> #}
+              <!-- {% if displayApprovedVisitorsCard %}
+                <li class="govuk-grid-column-one-quarter card-group__item">
+                  {{imageCard({ id: 'approvedVisitors', src: "/public/images/image_visitors.jpg", alt: "visitor picture", link: knownPages.profile.visitorsLink, description: "View your full list of approved visitors" }) }}
+                </li>
+              {%- endif %} -->
```

Comment:
Commented out to prevent access until views updated in an upcoming ticket and testing session has been arranged.